type-o-matic/Program.cs

```diff
@@ -172,7 +172,7 @@ static CSFile GenerateCSharpHashTableFromTypes (TypeAggregator types, PlatformNa
 		public static unsafe bool GetSwiftType (Type t, out SwiftMetatype md) {
 			using (var swiftStr = new SwiftString (t.FullName)) {
 				fixed (byte *swiftData = swiftStr.SwiftData) {
-					return GetSwiftType (swiftData, out md);
+					return SwiftCore.GetEnumMetadataByName (swiftData, out md);
```

Comment:
And then this can be simplified to:
```csharp
using (var swiftStr = new SwiftString (t.FullName)) {
 	return SwiftCore.GetEnumMetadataByName (swiftStr.SwiftData, out md);
}
```

Reply:
I put these changes in. Out of curiosity does it make a practical difference?

Reply:
No, at runtime I wouldn't expect much difference, the main advantage is simpler and shorter C# (also in my experience trying to avoid unsafe code whenever there's a good alternative leads to fewer bugs).